compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/QueryDesugar.java

```diff
@@ -147,7 +147,11 @@ BLangStatementExpression desugar(BLangQueryExpr queryExpr, SymbolEnv env) {
         if (queryExpr.isStream) {
             streamStmtExpr = ASTBuilderUtil.createStatementExpression(queryBlock, streamRef);
             streamStmtExpr.type = streamRef.type;
-        } else {
+        }
+//        else if (queryExpr.isTable) {
```

Comment:
I didn't complete this part. should a method be implemented for table functionality too like the 'buildStream' method?